label_studio/core/utils/schema/label_config_schema.json

```diff
@@ -17,6 +17,13 @@
         {"type": "boolean"},
         {"type": "number"}
       ]},
+    "@valueList": {
+      "anyOf": [
+      {
+        "type": "array",
```

Comment:
It should be string as well. It doesn't contain arrays (and actually can't as a property?), but only points to task data, the same as `value`

Reply:
Fixed.